src/get-started/install/_get-sdk.md

```diff
@@ -58,10 +58,13 @@ The following sections describe how to perform these tasks and finish the setup
 Once you have installed any missing dependencies, run the `flutter doctor` command again to
 verify that you’ve set everything up correctly.
 
-The `flutter` tool uses Google Analytics to anonymously report feature usage statistics
-and basic crash reports. This data is used to help improve Flutter tools over time.
-Analytics is not sent on the very first run or for any runs involving `flutter config`,
-so you can opt out of analytics before any data is sent. To disable reporting,
-type `flutter config --no-analytics` and to display the current setting, type
-`flutter config`. See Google's privacy policy:[www.google.com/intl/en/policies/privacy](https://www.google.com/intl/en/policies/privacy/).
-{: .alert-warning}
+<aside class="alert alert-warning" role="alert" markdown="1">
```

Comment:
I'm surprised you changed it from an alert warning to an aside.

Reply:
Visually, there is no change. Using an `<aside>` is just our convention (rather than just styling a plain paragraph). This is also what we do on www.

Reply:
Sorry: visually, there is a change: it looks better now. PTAL